src/transformers/models/flaubert/modeling_tf_flaubert.py

```diff
@@ -200,9 +200,8 @@ def get_masks(slen, lengths, causal, padding_mask=None):
 
     # sanity check
     # assert shape_list(mask) == [bs, slen]
-    if tf.executing_eagerly():
-        tf.debugging.assert_equal(shape_list(mask), [bs, slen])
-        assert causal is False or shape_list(attn_mask) == [bs, slen, slen]
+    tf.debugging.assert_equal(shape_list(mask), [bs, slen])
+    assert causal is False or shape_list(attn_mask) == [bs, slen, slen]
```

Comment:
Is this safe for a regular assert? Should this be converted to a debugging.assert or something?

Reply:
That's a good point -- if `shape_list(attn_mask)` contains a dynamic shape, then this assert will raise a `tf.errors.OperatorNotAllowedInGraphError`

Converting to `tf.debugging.assert_equal` just to be safe